newm-server-mobile/src/main/kotlin/io/projectnewm/server/mobile/MobileModule.kt

```diff
@@ -0,0 +1,14 @@
+package io.projectnewm.server.mobile
+
+import io.ktor.server.application.*
+import io.ktor.server.response.*
+import io.ktor.server.routing.*
+
+@Suppress("unused")
+fun Application.mobileModule() {
+    routing {
+        get("/mobile") {
+            call.respond(mapOf("Module" to "Mobile"))
```

Comment:
Just a placeholder to respond with `{ "Module":  "Mobile" }`

Reply:
So if I understand correctly, we're going to have different endpoints for mobile? Will these be reused if we end up making a web version of the app?

Reply:
Yes, we should be able to re-use.